### PR TITLE
Dispensing Form

### PR DESCRIPTION
**Pull request by solemabrothers**
https://app.asana.com/0/1133167201254915/1178249080093093

### CODE REVIEW

**Comment by dbaluku** [comment]
@solemabrothers 

- I don't think we need tabs since we have few brief sections they can appear on the same page 

- On the PoC Non-prescription section view, the encounter date and provider should be auto-filled and hidden 

- On edit of PoC completed dispensing encounter the data saved before does not return

File: omod/src/main/webapp/resources/htmlforms/Dispensing-form.xml [hunk 1]
```diff
@@ -52,104 +47,306 @@
...
-                    });
-                    if ($nonEmptyBoxes.length &gt; 0) {
-                        $nonEmptyBoxes.each(function () {
-                            $thisRow = jq(this).parents("div").show();
-                        });
+            $allRows = jq(this).find('.multi-drug');
+            $allRows.hide();
+            jq('#1-removeEntry').remove();
+            jq('#10-addEntry').remove();
+            jq('#1-toggleContainer').show();
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/Dispensing-form.xml:129]: are these additions or formatting effects.

File: omod/src/main/webapp/resources/htmlforms/Dispensing-form.xml [hunk 2]
```diff
@@ -52,104 +47,306 @@
...
+    </script>
+
 
     <style>
         .card .row {
             margin-bottom: 0px;
         }
     </style>
-    <div class="hidden">
-        <label>Encounter Date</label>
```
Comment (slubwama) [inline omod/src/main/webapp/resources/htmlforms/Dispensing-form.xml:81]: Why are you removing these block codes.

File: omod/src/main/webapp/resources/htmlforms/Dispensing-form.xml (hunk 1 above)
Comment (solemabrothers) [reply omod/src/main/webapp/resources/htmlforms/Dispensing-form.xml:129]: @slubwama yes they are

File: omod/src/main/webapp/resources/htmlforms/Dispensing-form.xml (hunk 2 above)
Comment (solemabrothers) [reply omod/src/main/webapp/resources/htmlforms/Dispensing-form.xml:81]: Because they must be displayed in retrospective entry @slubwama

**Comment by ssmusoke** [comment]
@solemabrothers What is the status of this PR, I assumed that it had already been merged

**Pull request closed without merging**